api/models.go

```diff
@@ -206,7 +214,32 @@ type MobileSearchResponseItem struct {
 	TagUid         *string    `json:"tagId,omitempty"`
 	Title          string     `json:"title"`
 	Image          *string    `json:"image,omitempty"`
-	Views          *int64     `json:"views,omitempty"`
+	ViewsType
+	Type           string     `json:"contentType"`
+	Date           *time.Time `json:"date,omitempty"`
+}
+
+type MobileFeedRequest struct {
+	MoreItems   	int         `json:"more_items"`
```

Comment:
is its number of items that you want to get? if yes have you anywhere limitations if it?

Reply:
not yet, maybe @YuriHechter knows what is the plan about this field?